src/components/dialogs/showUsers/ShowUsersDialog.tsx

```diff
@@ -0,0 +1,141 @@
+import React, {useState, useEffect} from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogActions from '@material-ui/core/DialogActions';
+import { DialogTitle } from '../addTodo/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TableFooter from '@material-ui/core/TableFooter';
+import Paper from '@material-ui/core/Paper';
+import { routes, pagination } from '../../../config/constants';
+import { Transition } from '../addTodo/styles';
+import { UsersData } from '../../../utils/interfaces/users';
+import { TodoCheckbox } from '../../checkbox/TodoCheckbox';
+import { TodoPagination } from '../../pagination/TodoPagination';
```

Comment:
Mojet prosto rename aninq `Pagination` and `Checkbox` qani ver 2-tx ogtagortumnq ?